File: 3 - Hermes' Final Word Of Wisdom/solution.js
function findLengthOfLastWord(sentence) {
    const words = sentence.trim().split(" ");
    return words.length === 0 ? 0 : words[words.length - 1].length;
}

process.stdin.resume();
process.stdin.setEncoding("utf-8");
let inputString = "";

process.stdin.on("data", (input) => {
    inputString += input;
});

process.stdin.on("end", () => {
    const inputLines = inputString.trim().split("\n");
    const N = parseInt(inputLines[0]);

    for (let i = 1; i <= N; i++) {
        const sentence = inputLines[i];
        console.log(findLengthOfLastWord(sentence));
    }
});

File: 3 - Hermes' Final Word Of Wisdom/solution.py
def find_length_of_last_word(sentence):
    words = sentence.strip().split()
    return len(words[-1]) if words else 0

if __name__ == "__main__":
    N = int(input())
    for _ in range(N):
        sentence = input().strip()
        print(find_length_of_last_word(sentence))